.travis.yml

```diff
@@ -56,6 +56,8 @@ compiler:
 install: 
   - if [[ "$CXX" = "clang++" ]]; then export CXX=clang++-3.6 CC=clang-3.6; fi
 before_script: 
+- wget -O doxygen https://cgal.geometryfactory.com/~mgimeno && sudo mv doxygen /usr/bin/doxygen && sudo chmod +x /usr/bin/doxygen
```

Comment:
I think the right URL is `https://cgal.geometryfactory.com/~mgimeno/doxygen/doxygen`.